client/my-sites/site-settings/disconnect-site/why/disconnect-reasons-menu.jsx

```diff
@@ -0,0 +1,99 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import SelectDropdown from 'components/select-dropdown';
+import PaginationFlow from './pagination-flow';
+import { isJetpackSite } from 'state/sites/selectors';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+import DisconnectReasonsSuggestion from './disconnect-reasons-suggestion';
+
+class DisconnectReasonsMenu extends Component {
+	constructor( props ) {
```

Comment:
After addressing all of my comments for the `constructor()` contents, you won't need to declare it at all.